front/app/containers/HomePage/SignedInHeader/HeaderImage.test.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import HeaderImage from './HeaderImage';
+import { mockHomepageSettings } from 'services/__mocks__/homepageSettings';
+import { getTheme } from '@citizenlab/cl2-component-library';
+import * as styledComponents from 'styled-components';
+import { THomepageBannerLayout } from 'services/homepageSettings';
+import { render, screen } from 'utils/testUtils/rtl';
+
+let mockHomepageBannerLayout: THomepageBannerLayout =
+  'full_width_banner_layout';
+
+jest.mock('hooks/useHomepageSettings', () =>
+  jest.fn(() => mockHomepageSettings(mockHomepageBannerLayout))
+);
+
+jest.spyOn(styledComponents, 'useTheme').mockReturnValue(getTheme());
```

Comment:
Why is this needed?

Reply:
Should still be deleted. I was getting an error about theme though 🤔

Reply:
Seems fine now ✅